controllers/datasciencecluster/kubebuilder_rbac.go

```diff
@@ -90,6 +90,11 @@ package datasciencecluster
 
 // +kubebuilder:rbac:groups="mcad.ibm.com",resources=appwrappers,verbs=create;delete;list;patch;get
```

Comment:
This can be removed.

Reply:
the reason to keep it is because https://github.com/red-hat-data-services/data-science-pipelines-operator/blob/6e215a50f9563f95c6b25f3979d6315bca739254/config/rbac/role.yaml#L160

Reply:
Ah right. It'll be possible to have it removed once opendatahub-io/data-science-pipelines-operator#347 is pulled downstream.